gnovm/pkg/gnopkgfetch/gnopkgfetch_testing.go

```diff
@@ -0,0 +1,99 @@
+package gnopkgfetch
```

Comment:
This is not a test file, so it's compiled into the binary. Please change it 🙏

Reply:
I'm not sure how to do it any other way, pls see https://github.com/gnolang/gno/pull/3123#discussion_r1854169892

Reply:
Can't you drop it now, after this fix you mentioned?

Reply:
refactored in 8d0a5a28da5a51dab22e7e8dfbea1f5ffbcb4a38 it's not compiled in